internal/cypress/docker/docker.go

```diff
@@ -206,9 +206,14 @@ func (handler *Handler) StartContainer(ctx context.Context, c cypress.Project) (
 		return nil, err
 	}
 
-	m, err := createMounts(files, pDir)
-	if err != nil {
-		return nil, err
+	var m []mount.Mount
```

Comment:
What if we initialize `m := []mount.Mount{}` at the beginning  and replace else block?

Reply:
For clarity, I preferred avoiding assigning twice the variable, if you think this is better the other way, I can change it

Reply:
I prefer not to have `else` conditions. It's up to you. Both ways work for me.